doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Authoring/DataSourceManagement/adding-database-driver-proc.adoc

```diff
@@ -0,0 +1,13 @@
+[id='adding-database-driver-proc_{context}']
+
+= Adding a database driver
+
+You can add a new database driver to {CENTRAL}.
+
+.Procedure
+. In {CENTRAL}, select the *Admin* icon in the upper-right corner of the screen and select *Data Sources*.
+. In the *DataSource Explorer* pane, click *Add Driver*.
++
+The *New driver* window opens.
+. In the *New driver* window, enter the name, driver class name, group ID, artifact ID, and version of the database driver.
```

Comment:
```suggestion
. In the *New driver* window, enter the *Name, Driver Class Name, Group Id, Artifact Id*, and *Version* of the database driver.
```

Reply:
I'd suggest to follow the same pattern as in other sections

Reply:
Okay, I will update the content accordingly.